src/check.h.in

```diff
@@ -987,6 +1000,33 @@ CK_DLL_EXP void CK_EXPORT srunner_run(SRunner * sr, const char *sname,
 
 
 /**
+ * Run a specific suite or test case or testcases with specific tags
```

Comment:
Slightly reworked your suggested text to make the interaction between the parameters and the env variables explicit.